src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,123 @@
+import classNames from "classnames";
+import { FILTER } from "../../types/filters";
+import { Link } from "react-router-dom"; 
+
+type Props = {
+  filterSex: FILTER,
+  handleSexFilterChange: (sex?: FILTER) => void,
+  query: string,
+  handleQueryChange: React.ChangeEventHandler<HTMLInputElement>,
+  centuries: string[],
+  handleCenturiesFilter: (ch: string) => void,
+  handleSelectAllCenturies: () => void,
+}
+
+export const PeopleFilters: React.FC<Props> = ({ 
+  filterSex, 
+  handleSexFilterChange,
+  query,
+  handleQueryChange,
+  centuries,
+  handleCenturiesFilter,
+  handleSelectAllCenturies,
+}) => {
+  const arrCenturies = [16, 17, 18, 19, 20];
+
+  return (
+    <nav className="panel">
+      <p className="panel-heading">Filters</p>
+
+      <p className="panel-tabs" data-cy="SexFilter">
+        <Link className={classNames({'is-active': filterSex === FILTER.ALL})} to="/people" 
```

Comment:
The `Link` component's `to` prop is set to `/people`. Ensure that this route is correctly handled in your router configuration to display the appropriate content when the link is clicked.